app/elements/code-editor/consoles/test-stats-console.html

```diff
@@ -5,6 +5,7 @@
 		:host{
 			display: block;
 			height: 100%;
+			overflow-x: scroll;
```

Comment:
Yes, but `auto` (so it will appear as necessary) is a valid option too. Why not use that?